### PR TITLE
Cart + order ajax and update

### PR DESCRIPTION
**Pull request by truongdh1903**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/???)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

### CODE REVIEW

**Comment by truongdh1903** [comment]
ready

File: app/views/carts/_cart.html.erb [hunk 1]
```diff
@@ -22,14 +29,14 @@
...
-    <input id="quantity" type="number" size="1" class="input-sm" title="Quantity" value="<%= cart["quantity"] %>" min="0" max="99" disabled>
     <div class="quantity buttons_added">
       <%= link_to "<button class='btn btn-primary btn-sm'>-</button>".html_safe,
         minus_qty_cart_path(cart["id"]),
         data: {params: "cost=#{@product_detail.cost}"},
         method: :put,
-        remote: true 
+        remote: true
       %>
+      <input id="quantity" type="number" size="1" class="input-sm" title="Quantity" value="<%= cart["quantity"] %>" min="0" max="99" disabled>
```
Comment (huongnt-2545) [inline app/views/carts/_cart.html.erb]: - I18n
- cho hằng số vào file settings

File: app/views/carts/_cart.html.erb [hunk 2]
```diff
@@ -1,7 +1,14 @@
 <tr class="cart_item" id="<%= "cart_#{cart["id"]}" %>">
+  <td class="product-check"> 
+    <%= link_to "<input type='checkbox' class='product-checkbox' #{'checked' if cart["checked"]}>".html_safe,
```
Comment (huongnt-2545) [inline app/views/carts/_cart.html.erb]: `"<input type='checkbox' class='product-checkbox' #{'checked' if cart["checked"]}>".html_safe` cho vào method helper để xử lý em nhé

File: app/views/static_pages/cart.html.erb [hunk 3]
```diff
@@ -16,21 +16,22 @@
...
-                    <th class="product-size">Size</th>
-                    <th class="product-price">Price</th>
-                    <th class="product-quantity">Quantity</th>
-                    <th class="product-subtotal">Total</th>
-                    <th class="product-remove">&nbsp;</th>
+                    <th class="product-thumbnail"><%= t ".product" %></th>
+                    <th class="product-color"><%= t ".color" %></th>
+                    <th class="product-size"><%= t ".size" %></th>
+                    <th class="product-price"><%= t ".price" %></th>
+                    <th class="product-quantity" width="20%"><%= t ".quantity" %></th>
```
Comment (huongnt-2545) [inline app/views/static_pages/cart.html.erb]: bê style vào file css đi em

File: app/views/static_pages/checkout.html.erb [hunk 4]
```diff
@@ -152,51 +105,36 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="col-md-4">
+        <br><br>
+        <div id="payment">
+          <ul class="payment_methods methods">
+            <li class="payment_method_bacs">
+              <input type="radio" data-order_button_text="" checked="checked" value="bacs" name="payment_method" class="input-radio" id="payment_method_bacs">
+              <label for="payment_method_bacs">Direct Bank Transfer </label>
```
Comment (huongnt-2545) [inline app/views/static_pages/checkout.html.erb]: I18n

File: app/views/static_pages/checkout.html.erb [hunk 5]
```diff
@@ -152,51 +105,36 @@
...
+              <div class="payment_box payment_method_bacs">
+                <p>Make your payment directly into our bank account. Please use your Order ID as the payment reference.</p>
```
Comment (huongnt-2545) [inline app/views/static_pages/checkout.html.erb]: I18n

**Comment by truongdh1903** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Pull request merged**